jenkinsapi/build.py

```diff
@@ -180,6 +184,23 @@ def _get_git_rev_branch(self):
     def _get_hg_rev_branch(self):
         raise NotImplementedError('_get_hg_rev_branch is not yet implemented')
 
+    def _get_git_repo_url(self):
+        # Sometimes we have None as part of actions. Filter those actions
+        # which have lastBuiltRevision in them
+        _actions = [x for x in self._data['actions']
+                    if x and "lastBuiltRevision" in x]
+        # old Jenkins version have key remoteUrl v/s the new version has a list remoteUrls
+        result = _actions[0].get("remoteUrls", _actions[0].get("remoteUrl"))
+        if isinstance(result, list):
+            result = result[0]
```

Comment:
what if I need to know all remote urls? Why not return list and let client decide?

Reply:
for a given repo you will only have one git url. Unless we are using multiscm plugin in which case the api will differ. I am not sure why we changed remoteurl to remoteUrls in current api version .. I looked at the dict it returns and it's a list with one value all the time.

Reply:
fixed
 if isinstance(result, list):
            result = ','.join(result)

Reply:
I changed the tests to assert Condition ... and the code coverage is back to red with that change .. but can you please look at it and merge it.